ext/mysqli/tests/mysqli_constants.phpt

```diff
@@ -10,6 +10,7 @@ mysqli.allow_local_infile=1
 $IS_MYSQLND = stristr(mysqli_get_client_info(), "mysqlnd");
 $constants = get_defined_constants(true);
 sort($constants);
+$IS_MARIADB = $constants['MYSQLI_IS_MARIADB'];
```

Comment:
That would need to be:
```suggestion
$IS_MARIADB = $constants["mysqli"]['MYSQLI_IS_MARIADB'];
```
But why is that necessary in the first place? I mean, why not use `MYSQLI_IS_MARIADB` directly below?